CHANGELOG.md

```diff
@@ -58,6 +58,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#7619](https://github.com/osmosis-labs/osmosis/pull/7619) Slight speedup/gas improvement to CL GetTotalPoolLiquidity queries
 * [#7622](https://github.com/osmosis-labs/osmosis/pull/7622) Create/remove tick events.
 * [#7623](https://github.com/osmosis-labs/osmosis/pull/7623) Add query for querying all before send hooks
+* [#7622](https://github.com/osmosis-labs/osmosis/pull/7622) Remove duplicate CL accumulator update logic.
+* [#7503](https://github.com/osmosis-labs/osmosis/pull/7503) Add IBC wasm light clients module.
+* [#7503](https://github.com/osmosis-labs/osmosis/pull/7677) feat(x/incentives): Restrict lockable durations to 2 weeks.
```

Comment:
This was merged incorrectly, this is repeating 7622 and 7503

Reply:
Fixed in the most recent commit